css/templates/land-see/page.css

```diff
@@ -210,7 +213,7 @@ h4,
+  background: transparent url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='104px' height='49px' viewBox='0 0 104 49'%3E%3Cdefs%3E%3C/defs%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,0.5c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,5.367c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,10.235c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,15.104c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,19.972c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,24.84c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,29.708c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,34.576c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,39.443c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,44.312c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3C/svg%3E") repeat-x left top local;
```

Comment:
can you use svgo and optimize this SVG , i think you will get some savings there

Reply:
Nice catch! That should give us some headroom on the 50 KB CSS limit. I think we may be able to bring this template home without exceeding. 🤞

Reply:
I’m also not opposed to loading this as a separate Svg image if this is the sole cause of lots of bytes

Reply:
Good to know. I'll keep that in mind.